Makefile

```diff
@@ -37,7 +39,21 @@ luacheck:
 test:
 	luatest -v --shuffle all:${SEED}
 
+$(LUACOV_STATS): test-coverage
```

Comment:
At the first glance the second launch should not happen, it does. A little strange.